src/main/java/com/bruno/models/Job.java

```diff
@@ -0,0 +1,92 @@
+package com.bruno.models;
+
+public class Job {
+
+	private Long id;
+	private String customerName;
+	private String address;
+	private String email;
+	private String phone;
+	private String date;
+	private String description;
+	
+	public Job() {
+		this.id = 0L;
+	}
+	
+	public Job(Long id, String customerName, String address, String email, String phone, 
+			String date, String description) {
+		this.id = id;
+		this.customerName = customerName;
+		this.address = address;
+		this.email = email;
+		this.phone = phone;
+		this.date = date;
+		this.description = description;
```

Comment:
Unless you're really using these, I suggest not defining a null/custom constructors and let Spring handle the default until you need to implement an interface-based copy constructor.